tests/unit/multieditor/test_actions.py

```diff
@@ -0,0 +1,911 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, print_function, division
+
+import json
+import os
+import pytest
+
+from inspirehep.modules.multieditor.actions import Addition, Deletion, Update, create_schema_record
+
+
+@pytest.fixture
+def get_schema():
+    curr_path = os.path.dirname(__file__)
+    with open(os.path.join(curr_path, 'fixtures/schema.json')) \
+            as data_file:
+        schema = json.load(data_file)
+    return schema
```

Comment:
Instead of using a whole schema, it's better to use snippets of it for the example records you use, and add a validation step for them on the same test, like https://github.com/inspirehep/inspire-next/blob/master/tests/unit/utils/test_utils_record.py#L36

Reply:
That way the test will only be affected if that part of the schema changes, and when it does, we will know right away (reducing variability and ensuring early detection).